monkey/monkey_island/cc/services/reporting/aws_exporter.py

```diff
@@ -7,7 +7,7 @@
 
 from common.cloud.aws_instance import AwsInstance
 from monkey_island.cc.environment.environment import load_server_configuration_from_file
-from monkey_island.cc.resources.exporter import Exporter
+from cc.services.reporting.exporter import Exporter
```

Comment:
Sure this works? 
Even if so, might be better to use full path.

Reply:
Fixed